examples/raw_stream_json_each_row.rs

```diff
@@ -0,0 +1,54 @@
+use std::str::from_utf8;
```

Comment:
Another real use case is to collect everything into one `Bytes` (or `Vec<u8>`). It's nice to have such an example, too!